docs/teams/view-team.md

```diff
@@ -0,0 +1,23 @@
+---
+title: View a Team
+intro: In this section, you will learn how to manage your team.
+links:
+  overview:
+  quickstart:
+  previous: teams/list-teams
+  next: teams/invitations/add-invitations
+  guides:
+  related:
+    - teams/add-teams
+    - teams/list-teams
+---
+
+1. On the selected project's page, locate the Teams card.
+2. Click View all on the Teams card to see the list of existing teams.
+   If there are no teams yet, create one by clicking the Add team button.
```

Comment:
If you want this "If there are no teams ..." to be displayed as a note you need to apply the proper `Markdown` syntax `>`, as can be seen in [1] and [2]

[1] https://www.devopness.com/docs/networks/add-network
[2] https://github.com/devopness/devopness/blob/main/docs/networks/add-network.md?plain=1#L22

```suggestion
    > If there are no teams yet, create one by clicking the Add team button.
```